symplyphysics/laws/thermodynamics/gas_pressure_from_temperature.py

```diff
@@ -0,0 +1,43 @@
+from sympy import (Eq, solve)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## The gas standard_pressure at a certain temperature depends on the temperature, standard standard_pressure and thermal coefficient.
+
+## Law is: p = p0 * y * T, where
```

Comment:
I am quite sure it is not always true. Write conditions when this law is applicable and give a link where you found this law.

Reply:
In your link the result is delta p, ie change of pressure.